### ISSUE TITLE
Publish version for pull request 660

### ISSUE THREAD

**Issue by aaronpkahn** [rpush/rpush#665]
**Describe the bug**
We are looking to use this commit https://github.com/rpush/rpush/pull/660 for our FCM HTTP transition this week.  Please publish a new gem version!

**To Reproduce**
1. Compare commits on master to published gem versions

**Expected behavior**
A new gem version is published. Maybe 7.1.0?

**Comment by aried3r** [comment]
@aaronpkahn, I'm planning to release a new version once #664 is merged and we get a green test run again.

If it will help, I am willing to drop support for older Rails and Ruby versions. Any particular version you need supported?

**Comment by aaronpkahn** [comment]
We currently run Rails 7.1.1 and Ruby 3.2.4

**Comment by aaronpkahn** [comment]
@aried3r checking in, do you need anything else from me?

**Comment by justindossey** [comment]
Is this going to happen soon? I'm seeing notifications failing as Google shuts down the deprecated APIs.

**Comment by requiemformemories** [comment]
I would like to ask if there have been any updates on this issue. Google is ramping down the deprecated API, and it might be completely shut down.

**Comment by benlangfeld** [comment]
https://github.com/rpush/rpush/pull/673 should move this forward.

**Comment by benlangfeld** [comment]
Anyone who can test with the current master branch would be of great assistance to ensure that a release will succeed.

**Comment by benlangfeld** [comment]
@aried3r I think we can release this now. Could you do the honours?

**Comment by aried3r** [comment]
@benlangfeld Thank you so much for all the work you put in!

I will try to have a look and publish a release this week.

**Comment by ghillahill** [comment]
I think the error we are getting:

> [ERROR] Rpush::DeliveryError, Unable to deliver notification 19929, received error (Failed to deliver to all recipients. Errors: DeprecatedApi.)


is related to this state?

**Comment by benlangfeld** [comment]
@ghillahill You need to provide more information. Code, more logs and rpush version.

**Comment by benlangfeld** [comment]
Closing this as a duplicate of https://github.com/rpush/rpush/issues/588 just to keep the issue tracker clean.

**Comment by benlangfeld** [comment]
@aried3r If you're short on time, please consider giving me access to push a release so as to stem the flow of confused complaints about neither GCM nor FCM working a-la #683 . I have work time set aside for bringing rpush up to scratch and a release is also blocking #680, #679, #672 and soon #675 .

`gem owner --add ben@langfeld.me rpush`

**Comment by aried3r** [comment]
@benlangfeld, check your mail. I'm out for today, but let me know if anything else is needed for a release.

/ping @ileitch for visibility. Ben has been very actively maintaining `rpush` and `modis` these last years. I believe he has earned our trust and I do currently not have the capacity to push development forward. I hope you are okay with adding another owner to `rpush`.
